src/app-layer-parser.c

```diff
@@ -964,6 +964,8 @@ int AppLayerParserParse(ThreadVars *tv, AppLayerParserThreadCtx *alp_tctx, Flow
                    alstate, AppLayerGetProtoName(f->alproto));
     }
 
+    uint64_t p_tx_cnt = AppLayerParserGetTxCnt(f->proto, alproto, f->alstate);
```

Comment:
you probably need a 'if (AppLayerParserProtocolIsTxAware(f->proto, alproto)) {' here?